scripts/openmc-track-to-vtk

```diff
@@ -74,26 +74,25 @@ def main():
             coords = h5py.File(fname).get('coordinates')
             n_points = coords.shape[0]
             for i in range(n_points):
-                points.InsertNextPoint(coords[i,:])
-                
+                points.InsertNextPoint(coords[i, :])
+
         # Create VTK line and assign points to line.
         line = vtk.vtkPolyLine()
         line.GetPointIds().SetNumberOfIds(n_points)
         for i in range(n_points):
             line.GetPointIds().SetId(i, point_offset+i)
-        
+
         cells.InsertNextCell(line)
         point_offset += n_points
     data = vtk.vtkPolyData()
     data.SetPoints(points)
     data.SetLines(cells)
-    
+
     writer = vtk.vtkXMLPPolyDataWriter()
-    writer.SetInput(data)
+    writer.SetInputData(data)
```

Comment:
This doesn't work for me with my outdated VTK.  I think the version of VTK I have is the one you get with `sudo apt-get install python-vtk` which we should try to support.  What do you think about changing this to

```python
    if (int(vtk.vtkVersion.GetVTKVersion().split('.')[0]) > 5):
        writer.SetInputData(data)
    else:
        writer.SetInput(data)
```